Tribute/Tribute.swift

```diff
@@ -133,21 +146,25 @@ extension Attributes {
         }
         self.kern = attributes[NSKernAttributeName] as? Float
         self.obliqueness = attributes[NSObliquenessAttributeName] as? Float
+        
         if let paragraph = attributes[NSParagraphStyleAttributeName] as? NSParagraphStyle {
-            let defaultParagraph = NSParagraphStyle.defaultParagraphStyle()
-            self.alignment = (paragraph.alignment == defaultParagraph.alignment) ? nil : paragraph.alignment
-            self.leading = (paragraph.lineSpacing == defaultParagraph.lineSpacing) ? nil : Float(paragraph.lineSpacing)
-            self.lineHeightMultiplier = (paragraph.lineHeightMultiple == defaultParagraph.lineHeightMultiple) ? nil : Float(paragraph.lineHeightMultiple)
-            self.paragraphSpacingAfter = (paragraph.paragraphSpacing == defaultParagraph.paragraphSpacing) ? nil : Float(paragraph.paragraphSpacing)
-            self.paragraphSpacingBefore = (paragraph.paragraphSpacingBefore == defaultParagraph.paragraphSpacingBefore) ? nil : Float(paragraph.paragraphSpacingBefore)
-            self.headIndent = (paragraph.headIndent == defaultParagraph.headIndent) ? nil : Float(paragraph.headIndent)
-            self.tailIndent = (paragraph.tailIndent == defaultParagraph.tailIndent) ? nil : Float(paragraph.tailIndent)
-            self.firstLineHeadIndent = (paragraph.firstLineHeadIndent == defaultParagraph.firstLineHeadIndent) ? nil : Float(paragraph.firstLineHeadIndent)
-            self.minimumLineHeight = (paragraph.minimumLineHeight == defaultParagraph.minimumLineHeight) ? nil : Float(paragraph.minimumLineHeight)
-            self.maximumLineHeight = (paragraph.maximumLineHeight == defaultParagraph.maximumLineHeight) ? nil : Float(paragraph.maximumLineHeight)
-            self.hyphenationFactor = (paragraph.hyphenationFactor == defaultParagraph.hyphenationFactor) ? nil : Float(paragraph.hyphenationFactor)
-            self.allowsTighteningForTruncation = (paragraph.allowsDefaultTighteningForTruncation == defaultParagraph.allowsDefaultTighteningForTruncation) ? nil : paragraph.allowsDefaultTighteningForTruncation
+            self.paragraph = paragraph
```

Comment:
I'd pass the paragraph as an argument to the `paraStyleCompare` instead of storing it as a property.
My thinking behind it is that you'll end up having a paragraph object and properties that contradict each other in terms of values. 
Also, from what I saw, you do not need it anywhere later, right?